pkg/controllers/manager.go

```diff
@@ -50,7 +50,10 @@ func (m *GenericControllerManager) RegisterControllers(ctx context.Context, cont
 		}
 	}
 	if err := m.AddHealthzCheck("healthz", healthz.Ping); err != nil {
```

Comment:
Can you call this livez (liveness)

Reply:
I can't tell if this is a joke. If it's not, the convention is healthz for liveness check